src/connection_manager.c

```diff
@@ -972,6 +972,8 @@ void qd_connection_manager_free(qd_connection_manager_t *cm)
         config_sasl_plugin_free(cm, saslp);
         saslp = DEQ_HEAD(cm->config_sasl_plugins);
     }
+
+    free(cm);
```

Comment:
this is a funny issue; dispatch does not free this, but because there is a global variable pointing to the current qd_router_t, it is not detected as a leak; when I want to start the router multiple times in a single process, subsequent initializations overwrite the global, and the leak gets reported